_src/pl/static/js/checkbox-replace.js

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
probably will need a js person to look this over as well - my js is pretty rusty.